Userland/Libraries/LibWeb/CMakeLists.txt

```diff
@@ -378,327 +370,25 @@ set(GENERATED_SOURCES
-libweb_js_wrapper(HTML/HTMLTableCellElement)
-libweb_js_wrapper(HTML/HTMLTableColElement)
-libweb_js_wrapper(HTML/HTMLTableElement)
-libweb_js_wrapper(HTML/HTMLTableRowElement)
-libweb_js_wrapper(HTML/HTMLTableSectionElement)
-libweb_js_wrapper(HTML/HTMLTemplateElement)
-libweb_js_wrapper(HTML/HTMLTextAreaElement)
-libweb_js_wrapper(HTML/HTMLTimeElement)
-libweb_js_wrapper(HTML/HTMLTitleElement)
-libweb_js_wrapper(HTML/HTMLTrackElement)
-libweb_js_wrapper(HTML/HTMLUListElement)
-libweb_js_wrapper(HTML/HTMLUnknownElement)
-libweb_js_wrapper(HTML/HTMLVideoElement)
-libweb_js_wrapper(HTML/ImageData)
-libweb_js_wrapper(HTML/MessageChannel)
-libweb_js_wrapper(HTML/MessageEvent)
-libweb_js_wrapper(HTML/MessagePort)
-libweb_js_wrapper(HTML/PageTransitionEvent)
-libweb_js_wrapper(HTML/PromiseRejectionEvent)
-libweb_js_wrapper(HTML/Storage)
-libweb_js_wrapper(HTML/SubmitEvent)
-libweb_js_wrapper(HTML/TextMetrics)
-libweb_js_wrapper(HTML/Worker)
-libweb_js_wrapper(HTML/WorkerGlobalScope)
-libweb_js_wrapper(HTML/WorkerLocation)
-libweb_js_wrapper(HTML/WorkerNavigator)
-libweb_js_wrapper(HighResolutionTime/Performance)
-libweb_js_wrapper(IntersectionObserver/IntersectionObserver)
-libweb_js_wrapper(NavigationTiming/PerformanceTiming)
-libweb_js_wrapper(RequestIdleCallback/IdleDeadline)
-libweb_js_wrapper(ResizeObserver/ResizeObserver)
-libweb_js_wrapper(SVG/SVGAnimatedLength)
-libweb_js_wrapper(SVG/SVGClipPathElement)
-libweb_js_wrapper(SVG/SVGDefsElement)
-libweb_js_wrapper(SVG/SVGElement)
-libweb_js_wrapper(SVG/SVGGeometryElement)
-libweb_js_wrapper(SVG/SVGGraphicsElement)
-libweb_js_wrapper(SVG/SVGCircleElement)
-libweb_js_wrapper(SVG/SVGEllipseElement)
-libweb_js_wrapper(SVG/SVGLength)
-libweb_js_wrapper(SVG/SVGLineElement)
-libweb_js_wrapper(SVG/SVGPathElement)
-libweb_js_wrapper(SVG/SVGPolygonElement)
-libweb_js_wrapper(SVG/SVGPolylineElement)
-libweb_js_wrapper(SVG/SVGRectElement)
-libweb_js_wrapper(SVG/SVGSVGElement)
-libweb_js_wrapper(SVG/SVGTextContentElement)
-libweb_js_wrapper(Selection/Selection)
-libweb_js_wrapper(UIEvents/FocusEvent)
-libweb_js_wrapper(UIEvents/KeyboardEvent)
-libweb_js_wrapper(UIEvents/MouseEvent)
-libweb_js_wrapper(UIEvents/UIEvent)
-libweb_js_wrapper(URL/URL)
-libweb_js_wrapper(URL/URLSearchParams ITERABLE)
-libweb_js_wrapper(WebSockets/WebSocket)
-libweb_js_wrapper(XHR/ProgressEvent)
-libweb_js_wrapper(XHR/XMLHttpRequest)
-libweb_js_wrapper(XHR/XMLHttpRequestEventTarget)
-
-invoke_generator(
-    "Enums"
-    Lagom::GenerateCSSEnums
-    "${CMAKE_CURRENT_SOURCE_DIR}/CSS/Enums.json"
-    ""
-    "CSS/Enums.h"
-    "CSS/Enums.cpp"
-    arguments -j "${CMAKE_CURRENT_SOURCE_DIR}/CSS/Enums.json"
-)
-
-invoke_generator(
-    "MediaFeatureID"
-    Lagom::GenerateCSSMediaFeatureID
-    "${CMAKE_CURRENT_SOURCE_DIR}/CSS/MediaFeatures.json"
-    ""
-    "CSS/MediaFeatureID.h"
-    "CSS/MediaFeatureID.cpp"
-    arguments -j "${CMAKE_CURRENT_SOURCE_DIR}/CSS/MediaFeatures.json"
-)
+setup_libweb_generators(LibWeb)
```

Comment:
It will be added later when we need to distinguish between Serenity and Lagom, but yeah for now it's not used. I've removed it here, and will add it back in the headless-browser PR!